src/components/layout/NavBar.jsx

```diff
@@ -1,28 +1,65 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import MediaQuery from 'react-responsive';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import NavButton from './NavButton';
+import { breakpoints } from '../../localconf';
 import './NavBar.less';
 
 /**
  * NavBar renders row of nav-items of form { name, icon, link }
  * @param { dictIcons, navTitle, navItems } params
  */
 class NavBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      menuOpen: false,
+    };
+  }
+
   componentDidMount() {
     this.props.onInitActive();
   }
 
-   isActive = id => {     
-    let toCompare = this.props.activeTab.split('/').filter(x => x !== 'dev.html').join('/');
+  isActive = (id) => {
+    const toCompare = this.props.activeTab.split('/').filter(x => x !== 'dev.html').join('/');
     return toCompare.startsWith(id);
   }
 
+  toggleMenu = () => {
+    this.setState(prevState => ({ menuOpen: !prevState.menuOpen }));
+  }
+
   render() {
+    const navItems = this.props.navItems.map(
+      (item, index) => (
+        (item.link.startsWith('http')) ?
+          <a className='nav-bar__link nav-bar__link--right' key={item.link} href={item.link}>
+            <NavButton
+              item={item}
+              dictIcons={this.props.dictIcons}
+              isActive={this.isActive(item.link)}
+              onActiveTab={() => this.props.onActiveTab(item.link)}
+              tabIndex={index + 1}
+            />
+          </a> :
+          <Link className='nav-bar__link nav-bar__link--right' key={item.link} to={item.link}>
+            <NavButton
+              item={item}
+              dictIcons={this.props.dictIcons}
+              isActive={this.isActive(item.link)}
+              onActiveTab={() => this.props.onActiveTab(item.link)}
+              tabIndex={index + 1}
+            />
+          </Link>
+      ));
+
     return (
       <div className='nav-bar'>
         <header className={`nav-bar__header ${this.props.isFullWidth ? 'nav-bar__header--full-width' : ''}`}>
```

Comment:
Very awesome work! I notice now the whole website is all full width, so could you also remove this `isFullWidth` logic since it's no longer useful? thanks!